tests/test_match.py

```diff
@@ -384,11 +384,6 @@ def test_known_fusions(self, conn, known_variant, related_variants):
         for variant in related_variants:
             assert variant in names
 
-    def test_known_fusion_single_gene_no_match(self, conn):
```

Comment:
do we no longer want this functionality?

Reply:
I don't think there's a case where this can pass now - it seems that a gene is always matched and, from my understanding, that is intentional.

Reply:
rather than remove the test we could use a different fusion partner. This should still work in prinicpal, even if  a BCR e.13 fusion currently exists. What match is it retrieving? could that be posted here in case it is incorrect?

Reply:
The tricky part is that it's not matching a positional variant, it's matching a very generic "BCR mutation" CategoryVariant (this one: https://graphkb.bcgsc.ca/view/164:915).

Based on this section of the code - https://github.com/bcgsc/pori_graphkb_python/blob/v1.7.0/graphkb/match.py#L541-L571 - that seems to be the intended behaviour.

Instead of BCR we can just change the test to a different gene - TERT seems to work. However, if anyone ever adds a "TERT mutation" variant to the database, the test will just start failing again.

@creisle and @elewis2 what do we think we should do here?

Reply:
Looks like we've already decided on updating this test to pass in https://github.com/bcgsc/pori_graphkb_python/pull/67 so I'm going to say this is resolved, for the purposes of this PR.